### PR TITLE
Feature: sizes customizing - set of new components

### PR DESCRIPTION
**Pull request by siarhei-epam**
### Description:
Added a set of new components with support for size customization.

LabledInput - changed right padding for labelPosition prop from 5px to 6px (agreed with the design team)
Paginator - changed width for 3 dots (agreed with the design team)
PickerInputBody - changed left/right padding for search container according design
DatePickerBody - changed top padding for container (agreed with the design team)
TextArea - fixed line-height in 24 size, changed internal paddings according design

### CODE REVIEW

File: uui/settings.ts [hunk 1]
```diff
@@ -167,5 +203,120 @@ export const settings = {
...
+                18: '18',
+                24: '24',
+                30: '30',
+                36: '36',
+                42: '42',
+                48: '48',
+                60: '48',
+            },
+            leftPadding: {
+                editable: '0',
```
Comment (AlekseyManetov) [inline uui/settings.ts:284]: hardcode editable value

File: uui/settings.ts [hunk 2]
```diff
@@ -167,5 +203,120 @@ export const settings = {
...
+                36: '18',
+                48: '18',
+                60: '18',
+            },
+            resizeMarker: {
+                default: '12',
+            },
+            leftPadding: {
+                default: '12',
+                utmost: '24',
```
Comment (AlekseyManetov) [inline uui/settings.ts:248]: Probably firstColumnPadding and lastColumnPading

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: epam-promo/components/typography/Text.tsx [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
     /**
      * Defines component color.
      * @default 'gray80'
      */
     color?: 'blue' | 'green' | 'amber' | 'red' | 'white' | 'gray5' | 'gray50' | 'gray60' | 'gray80' | 'gray90' | uui.TextProps['color'];
+    /**
+     * Defines text size
+     * @default '36'
+     */
+    size?: TextSize;
```
[the remark this answers is not in the document]
Comment (siarhei-epam) [reply epam-promo/components/typography/Text.tsx:16]: to be able to override the size type in the uui package, accordingly, this must be written in the skins

File: uui/components/pickers/DataPickerBody.module.scss [hunk 4]
```diff
@@ -1,8 +1,8 @@
 @use  '../../assets/styles/index' as *;
 
-:global(.uui-pickerInput-body) {
-    --uui-dt-cell-border-width: 0;
-    --uui-pickerInput-selected_mark: var(--uui-info-50);
+:global(.uui-picker_input-body) {
```
[the remark this answers is not in the document]
Comment (siarhei-epam) [reply uui/components/pickers/DataPickerBody.module.scss:3]: we didn't use camel case, I brought to the unity of using classes on the project

File: uui/components/tables/DataTable.tsx [hunk 5]
```diff
@@ -64,24 +79,9 @@ export interface DataTableProps<TItem, TId, TFilter = any> extends IEditable<Dat
      * Enables collapse/expand all functionality.
      * */
     showFoldAll?: boolean;
-
-    /**
-     * Defines table header size
-     * @default '36'
-     * */
-    headerSize?: '36' | '48' | '60';
```
[the remark this answers is not in the document]
Comment (siarhei-epam) [reply uui/components/tables/DataTable.tsx:72]: moved to DataTableMods to be able to override the type

File: uui/components/tables/DataTableHeaderCell.tsx [hunk 6]
```diff
@@ -16,78 +22,107 @@ import { ReactComponent as OpenedDropdownIcon } from '@epam/assets/icons/navigat
...
                     color="inverted"
                     content={ renderTooltip(this.props.column) }
                     cx={ css.cellTooltip }
                     openDelay={ 600 }
                 >
-                    <Text key="text" lineHeight="30" fontSize="14" size="30" cx={ cx(css.caption, this.getTextStyle(), uuiDataTableHeaderCell.uuiTableHeaderCaption) }>
-                        {this.props.column.caption}
+                    <Text
+                        key="text"
+                        lineHeight={ settings.sizes.dataTable.header.row.cell.columnCaption.lineHeight as TextProps['lineHeight'] }
```
Comment (AlekseyManetov) [inline uui/components/tables/DataTableHeaderCell.tsx:82]: Probably we can move `settings.sizes.dataTable.header.row.cell.columnCaption.lineHeight` from settings to css variable?

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: uui/components/pickers/PickerItem.tsx [hunk 7]
```diff
@@ -59,8 +59,8 @@ export function PickerItem<TItem, TId>(props: PickerItemProps<TItem, TId>) {
         <FlexCell width="auto" cx={ [css.root, 'uui-typography', cx] }>
             <FlexRow
                 size={ itemSize as FlexRowProps['size'] }
-                cx={ isMultiline && [css.multiline, css[`vertical-padding-${itemSize}`]] }
-                columnGap="12"
+                cx={ [isMultiline && css.multiline, css.columnGap] }
+                rawProps={ { style: { '--uui-picker_item-vertical-padding': `${settings.sizes.pickerInput.body.dropdown.row.cell.item.verticalPadding[itemSize]}px` } as React.CSSProperties } }
```
[the remark this answers is not in the document]
Comment (siarhei-epam) [reply uui/components/pickers/PickerItem.tsx:63]: Here we need to dive a little deeper into this, collect the components and see how we can generalize

**Comment by NatalliaAlieva** [comment]
Column header no longer wraps to 2 lines in the "Condensed view" example.

**Comment by NatalliaAlieva** [comment]
Navigation-chevron_right icon larger on Dev than on Prod.
Screenshot 2024-09-25 at 15 02 18

**Pull request merged**